tests/performance/run_server_performance_test.py

```diff
@@ -0,0 +1,279 @@
+import argparse
+import csv
+import datetime
+import random
+import subprocess
+import threading
+import time
+from collections import defaultdict
+
+
+VERBOSE = False
+
+
+def timeit(func):
```

Comment:
Don't we have similar functionality somewhere? Wouldn't it be better to put that to a common place and use it?

Reply:
The name collision might be a bit annoying. This decorator does something else.

Reply:
Maybe it would be better to use a slightly different name? (So go to definition works better witihin IDEs).